client/src/components/modals/pick-request-type.vue

```diff
@@ -11,7 +11,7 @@
             <v-simple-table class="text-left">
           <tr v-for="type in tableData.col1" :key="type.value+'-tr'">
             <td class="clickable-cell" :id="type.value" @click="chooseType(type)">
-              <v-tooltip bottom max-width="500" open-delay="500">
```

Comment:
Was this an intentional change? I don't see it described in the PR description or ticket description.

Reply:
Yes. It's mentioned in invisio (Under Tool Tip notes)